packages/core/src/controllers/subscriber.ts

```diff
@@ -40,7 +41,7 @@ export class Subscriber extends ISubscriber {
   private pendingSubscriptionWatchLabel = "pending_sub_watch_label";
   private pendingSubInterval = 20;
   private storagePrefix = CORE_STORAGE_PREFIX;
-
+  private subscribeTimeout = 10_000;
```

Comment:
This looks similar to `private publishTimeout = 10_000;` I saw previously. If they are aimed to be the same, perhaps it would be better to move them into a shared const file?